homeworks/src/main/java/hw40/entity/Order.java

```diff
@@ -0,0 +1,36 @@
+package hw40.entity;
+
+import jakarta.persistence.*;
+import lombok.*;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+@NoArgsConstructor
+@Getter
+@Setter
+@EqualsAndHashCode(of = "id")
+@ToString
+@Entity
+@Table(name = "orders", schema = "postgres")
+public class Order {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Temporal(TemporalType.TIMESTAMP)
+    @Column(name = "date", nullable = false)
+    private LocalDateTime date;
+
+    @Column(name = "cost")
+    private Double cost;
+
+    @OneToMany(cascade = CascadeType.ALL, fetch = FetchType.LAZY, mappedBy = "order")
```

Comment:
Putting cascade type all here is a bad idea, because if you decide deleting an order, then you'll delete products associated with it as well. I think its not the desired behavior. Change cascade type to PERSIST, MERGE, REFRESH